apps/judicial-system/backend/src/app/modules/case/case.service.ts

```diff
@@ -472,6 +474,23 @@ export class CaseService {
       })
   }
 
+  private async deliverCaseToPolice(theCase: Case): Promise<boolean> {
+    const pdf = await getCourtRecordPdfAsString(theCase, this.formatMessage)
```

Comment:
```suggestion
    await this.refreshFormatMessage()
    const pdf = await getCourtRecordPdfAsString(theCase, this.formatMessage)
```

Reply:
The refresh is actually done a bit earlier in the call chain, i.e. in the public method `deliver`. That method calls a number of `deliver...` methods some of which use `formatMessage`. So I don't think a refresh is necessary here.